ui/lib/kv/addon/components/page/secret/overview.hbs

```diff
@@ -0,0 +1,102 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<KvPageHeader @breadcrumbs={{@breadcrumbs}} @pageTitle={{@path}}>
+  <:tabLinks>
+    <li>
+      <LinkTo @route="secret.index" @models={{array @backend @path}} data-test-secrets-tab="Overview">Overview</LinkTo>
+    </li>
+    <li>
+      <LinkTo @route="secret.details" @models={{array @backend @path}} data-test-secrets-tab="Secret">Secret</LinkTo>
+    </li>
+    <li>
+      <LinkTo
```

Comment:
Great question! We don't. I think we have the page handle hide/show logic more granularly because the details in the `Metadata` tab can be gleaned from `custom_metadata` from the secret OR `@metadata`. Plus the toolbar has delete/edit actions that technically a user could still perform without `READ metadata`